### ISSUE TITLE
undefined symbol: TIFFFieldDataType, version LIBTIFF_4.0

### ISSUE THREAD

**Issue by jiangyijin** [cvg/limap#67]
When I run python -c "import limap", I encounter the error "Undefined symbol: TIFFFieldDataType, version LIBTIFF_4.0" on Ubuntu 20.04.
(pytorch) ubuntu@ml-ubuntu20-04-desktop-v1-0-108gb-100m:~/Desktop/limap$ python -c "import limap"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ubuntu/Desktop/limap/limap/__init__.py", line 3, in <module>
    from _limap import *
ImportError: /lib/x86_64-linux-gnu/libfreeimage.so.3: undefined symbol: TIFFFieldDataType, version LIBTIFF_4.0

Then I tried the solution from this GitHub issue: https://github.com/colmap/colmap/issues/1937 by using conda uninstall libtiff to resolve it. However, I faced another issue that has been bothering me all day.
(pytorch) ubuntu@ml-ubuntu20-04-desktop-v1-0-108gb-100m:~/Desktop/limap$ python -c "import limap"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ubuntu/Desktop/limap/limap/__init__.py", line 6, in <module>
    from . import point2d
  File "/home/ubuntu/Desktop/limap/limap/point2d/__init__.py", line 1, in <module>
    from .superpoint import *
  File "/home/ubuntu/Desktop/limap/limap/point2d/superpoint/__init__.py", line 1, in <module>
    from .main import run_superpoint
  File "/home/ubuntu/Desktop/limap/limap/point2d/superpoint/main.py", line 12, in <module>
    from hloc import extract_features
  File "/home/ubuntu/Desktop/limap/third-party/Hierarchical-Localization/hloc/extract_features.py", line 12, in <module>
    import PIL.Image
ModuleNotFoundError: No module named 'PIL.Image'
I am using CUDA 11.8, torch 1.13.1, colmap3.8.

**Comment by B1ueber2y** [comment]
Hi. Maybe this is relevant: https://github.com/cvg/limap/issues/30. Have you tried this? Thanks!

**Comment by jiangyijin** [comment]
> Hi. Maybe this is relevant: #30. Have you tried this? Thanks!嗨这可能是一个问题：#30。你试过这个吗？谢谢你，谢谢

Hello. Thank you for your response. I resolved the 'Undefined symbol: TIFFFieldDataType, version LIBTIFF_4.0' issue by using 'conda uninstall libtiff' with #30. However, I then encountered a 'ModuleNotFoundError: No module named 'PIL.Image''. I resolved this problem by using 'pip install torchvisio' and 'pip install Pillow'. Thank you very much for your reply.
